stackslib/src/main.rs

```diff
@@ -26,10 +26,10 @@ extern crate stacks_common;
 #[macro_use(o, slog_log, slog_trace, slog_debug, slog_info, slog_warn, slog_error)]
 extern crate slog;
 
-#[cfg(not(target_env = "msvc"))]
+#[cfg(not(any(target_os = "macos", target_os = "windows", target_arch = "arm")))]
```

Comment:
I'd like to give jemalloc its own feature flag, but that can be a future PR

Reply:
fully agreed there - we can adjust this diff later